crates/libs/windows/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg_attr(docsrs, doc = "This is a stub. The latest API documentation is here: <https://microsoft.github.io/windows-docs-rs/>")]
+#![cfg_attr(docsrs, doc = "")]
 /*!
 Learn more about Rust for Windows here: <https://github.com/microsoft/windows-rs>
 */
```

Comment:
```suggestion
/*!
This is a stub. The latest API documentation is here: <https://microsoft.github.io/windows-docs-rs/>

Learn more about Rust for Windows here: <https://github.com/microsoft/windows-rs>
*/
```

Thanks Chris, I think the same effect can be achieved without messing with docsrs flags/attributes. No?

Reply:
Sure. But wouldn't that also affect the docs generated for microsoft.github.io? Which is not a stub.

Reply:
Oh, got it. 😊